src/cocaine-app/cache_transport/__init__.py

```diff
@@ -20,8 +20,11 @@
 
 
 def encode_dict(params):
-    return dict([(k, v if not isinstance(v, unicode) else v.encode('utf-8'))
-                 for k, v in params.iteritems()])
+    return {
+        k: v
+        if not isinstance(v, unicode) else v.encode('utf-8')
```

Comment:
This hides that `if/else` is about only `v`, how about to use lambda or function here, for example:
```python
def encode = lambda v:
    v if not isinstance(v, unicode) else v.encode('utf-8')

return {
    k: encode(v)
    for k, v in params.iteritems()
}
```